src/pages/patientView/clinicalInformation/lib/clinicalAttributesStyleConsts.js

```diff
@@ -0,0 +1,285 @@
+        order: 2,
+    },
+    BREAST_CANCER_RECEPTOR_STATUS: {
+        order: 6,
+    },
+    CANCER_TYPE: {
+        primary: true,
+        order: 3,
+    },
+    CANCER_TYPE_DETAILED: {
+        primary: false,
+        order: 4,
+    },
+    DERIVED_NORMALIZED_CASE_TYPE: {
+        primary: true,
+        values: {
+            Primary: colors.sample_color_primary,
+            Progressed: colors.sample_color_recurrence,
+            Recurrence: colors.sample_color_recurrence,
+            Metastasis: colors.sample_color_metastasis,
+        },
+        order: 1,
+    },
+    DERIVED_SAMPLE_LOCATION: {
+        primary: false,
+        order: 2,
+    },
+    DFS_STATUS: {
+        primary: true,
+        values: {
+            _: '#f00',
+            DiseaseFree: 'rgb(0, 128, 0)',
+            Yes: 'rgb(0, 128, 0)',
+        },
+        order: 9,
+    },
+    DFS_MONTHS: {
+        primary: false,
+        formatter: t => {
+            return `${t} months`;
+        },
+        order: 10,
+    },
+    DRIVER_MUTATIONS: {
+        order: 6,
+    },
+    ERG_FUSION_ACGH: {
+        primary: true,
+        formatter: t => {
+            return `ERG-fusion aCGH: ${t}`;
+        },
+        order: 6,
+    },
+    ETS_RAF_SPINK1_STATUS: {
+        order: 6,
+    },
+    GLEASON: {
+        primary: true,
+        formatter: t => {
+            return `Gleason: ${t}`;
+        },
+    },
+    GLEASON_SCORE: {
+        order: 6,
+    },
+    GENDER: {
+        order: 1,
+    },
+    HISTOLOGY: {
+        order: 6,
+    },
+    KNOWN_MOLECULAR_CLASSIFIER: {
+        order: 5,
+    },
+    MOUSE_STRAIN: {
+        order: 6,
+    },
+    PATIENT_DISPLAY_NAME: {
+        primary: false,
+        order: 0,
+    },
+    OS_MONTHS: {
+        primary: false,
+        formatter: t => {
+            return `${t} months`;
+        },
+        order: 8,
+    },
+    OS_STATUS: {
+        primary: true,
+        values: {
+            DECEASED: '#f00',
+            DEAD: '#f00',
+            LIVING: 'rgb(0, 128, 0)',
+            ALIVE: 'rgb(0, 128, 0)',
+        },
+        order: 7,
+    },
+    SAMPLE_DISPLAY_NAME: {
+        primary: false,
+        order: 0,
+    },
+    SERUM_PSA: {
+        primary: true,
+        formatter: t => {
+            return `Serum PSA: ${t}`;
+        },
+        order: 6,
+    },
+    SEX: {
+        order: 1,
+    },
+    TMPRSS2_ERG_FUSION_STATUS: {
+        primary: true,
+        formatter: t => {
+            return `TMPRSS2-ERG Fusion: ${t}`;
+        },
+        order: 6,
+    },
+    TUMOR_GRADE: {
+        order: 6,
+    },
+    TUMOR_STAGE_2009: {
+        primary: true,
+        order: 6,
+    },
+};
+
+/**
+ * Functions for ordering and styling clinical attributes.
+ */
+/**
+ * Order patient clinical attributes by the object mapping config.
+ * If the key does not exist in config, then it shall be ordered last.
+ * @param {string} key
+ * @param {object} config
+ */
+export const getOrder = (key, config) => {
```

Comment:
```suggestion
export function getOrder(key, config){
```
Let's declare this function like we do everywhere else. Makes it a bit faster to differentiate between a function being exported and an object being exported.

Reply:
Same goes for all other top level func decls